Source/DafnyCore/Verifier/Translator.cs

```diff
@@ -7233,7 +7233,7 @@ public ForceCheckToken(IToken tok)
       Contract.Ensures(Contract.Result<Bpl.Ensures>() != null);
 
       Bpl.Ensures ens = new Bpl.Ensures(ForceCheckToken.Unwrap(tok), free, condition, comment);
-      ens.Description = new PODesc.AssertStatement(errorMessage ?? "This is the postcondition that could not be proven.");
+      ens.Description = new PODesc.AssertStatement(errorMessage ?? "this is the postcondition that could not be proven.");
```

Comment:
If you are going with the lower case style, then remove the ending periods as well

Reply:
It is usually in a full sentence, though, of the form "Error: here is something in lowercase."

Reply:
Yes, but the style in dafny elsewhere is predominantly to start with lower-case and not have a period, except when there are clearly multiple sentences. I'm not wedded to this style, and am open to a team discussion, but I'd like it to be consistent.

Reply:
Ah, good point. I'm fine with leaving out the periods.